routes/main/homepage.js

```diff
@@ -65,30 +18,10 @@ module.exports = function(config, req, res) {
     mainURL: env.get("NODE_ENV") === "development" ? "/scripts/main.js" : "/dist/main.js"
```

Comment:
The homepage is doing a requirejs build step too?  If so, it won't point to the same set of `main.js` files, so this seems wrong.